src/display/api_compatibility.js

```diff
@@ -42,9 +44,15 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC')) {
       compatibilityParams.disableStream = true;
     }
   })();
+
+  // Support: Node.js
+  (function checkFontFaceAndImage() {
+    // Node.js is missing native support for `@font-face` and `Image`.
+    if (isNodeJS()) {
+      compatibilityParams.disableFontFace = true;
```

Comment:
Now that this is set in the library, I think we can simplify:

https://github.com/mozilla/pdf.js/blob/57e8676180752c375e0d364234565d6240a9a3ec/examples/node/pdf2png/pdf2png.js#L62-L66

to:

```javascript
pdfjsLib.getDocument({ data: rawData, }).then(function (pdfDocument) {
```

Reply:
> [...] so please check if the compatibility introduced here is indeed included in the library bundle.

Given that https://github.com/mozilla/pdf.js/blob/master/src/display/api_compatibility.js is imported by the API, this should be totally fine; see also https://github.com/mozilla/pdf.js/blob/2921cc0eba053896860e62a4bd1ffb5d29be51e5/src/display/api.js#L29

Furthermore, that fact that the `document` errors are now gone from the Travis output also shows that including these new Node.js specific compatibility options works fine.